packages/webapp/src/components/ErrorHandler/PureReactErrorFallback/styles.module.scss

```diff
@@ -0,0 +1,149 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+@import '../../../assets/mixin';
+
+.container {
+  min-height: 100vh;
+  min-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.background {
+  position: fixed;
+  background-color: var(--mainBackground, #fafcfb);
+  top: 56px;
+  z-index: -1;
+
+  @include xs-breakpoint {
+    top: 0px;
+    width: 100vw;
+    padding-block: 48px;
+  }
+}
+
+.logo {
+  height: 100px;
+  width: auto;
+  margin-block: 48px;
+  margin-left: 10%;
+  align-self: flex-start;
+
+  @include xs-breakpoint {
+    height: 56px;
+    margin: 32px;
+  }
+}
+
+.textContainer {
+  width: 90%;
+  max-width: 648px;
+  background-color: var(--mainBackground, #fafcfb);
+  padding: 32px;
+
+  p {
+    font-family: 'Open Sans';
+    line-height: normal;
+    font-style: normal;
+    font-size: 18px;
+    color: var(--grey900, #282b36);
+    letter-spacing: -0.396px;
+  }
+
+  @include xs-breakpoint {
+    width: 100%;
+    padding-bottom: 56px;
+
+    p {
+      font-size: 14px;
+      letter-spacing: -0.308px;
+    }
+  }
+}
+
+.title {
+  color: #16423d; /* Figma's teal 900 on this page is not is not the app's teal 900 -- this should be checked */
```

Comment:
The app's var is close enough that we could probably use that one?

Reply:
(Discussed today in tech daily but just noting here for GitHub too)...

This colour is part of the new design system; Loïc provided this really helpful [Figma link](https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?type=design&node-id=1-60)). Since we're going to move towards using CSS variables according to that convention, we'll eventually move the other way and migrate the old teal over when ready.